index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="./favicon.ico" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" type="text/css" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Adebowale - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <a href="" class="john-doe zeeh">ADEBOWALE</a>
+        <nav class="header-bar">
+          <a href="" class="openNav"><i class="fa fa-bars cursor"></i></a>
+        </nav>
+      </nav>
+      <!-- mobile Navigation Menu -->
+
+      <div class="mobile-nav" id="myLinks">
+        <div class="close-but">
+          <a href="" class="menu-list">&times;</a>
+        </div>
+        <br /><br />
+        <a href="#portfolio" class="portfolio-link">Portfolio</a>
+        <div class="line-bar">
+          <img src="images/Separtor_FullWidthBottom.png" alt="line logo" />
+        </div>
```

Comment:
- I have noticed that the line bar `div` between the links is not working and it's not consistent with the Figma design. To align with the design specifications, could you kindly adjust the `div` to include a line between the links? 💯

### Your mobile menu:
  <img src="https://user-images.githubusercontent.com/107952724/233194318-1e9f2b11-8bde-467f-b6bb-0f787b20f7bb.png" width="25%" />

### Figma mobile menu:
  <img src="https://user-images.githubusercontent.com/107952724/233202842-852f70fb-86af-417e-90fe-ed6fcaef6fe8.png" width="25%" />